Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml

```diff
@@ -2,7 +2,7 @@
   name: Gestio
   parent: BaseWeaponRifle
   id: WeaponRifleGestio
-  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.   Uses .20 rifle ammo.
+  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.   Uses .30 rifle ammo.
```

Comment:
French spacing.
```suggestion
  description: An old prototype burst-fire NanoTrasen marksman rifle. Manufactured by Silver Industries. It hasn't ever left the trials. Given it's antiquity it is considered a civilian grade weapon.  Uses .30 rifle ammo.
```

Reply:
While you're at it, fix the grammar too: "Given <ins>its</ins> antiquity"